src/textual/widgets/_markdown.py

```diff
@@ -146,10 +147,14 @@ def build_from_token(self, token: Token) -> None:
         self._token = token
         style_stack: list[Style] = [Style()]
         content = Text()
+        repeating_whitespace = re.compile(r"[ \t]+")  # Space or tabs get reduced.
```

Comment:
Wouldn't `"\s+"` included spaces and tabs?

There isn't much benefit in compiling the regex here. The `re` module has its own cache. I'd suggest moving the `re.compile` to module scope, or omitting and and let the `re` module do the caching.

Reply:
> Wouldn't "\s+" included spaces and tabs?

I didn't see any benefit in matching line breaks. Should we?

Reply:
I think linebreaks are converted to spaces anyway in markdown. There are also a few other whitespace characters covered by `s`

Reply:
In that case I've dropped it to just that.